query_test.go

```diff
@@ -33,7 +35,8 @@ func TestClient_Query_Exec_allTypes(t *testing.T) {
 		2,                        // intvalue
 		3,                        // smallintvalue
 		5,                        // tinyintvalue
-		&timeUUID,                // timeuuidvalue
+		big.NewInt(0).Sub(big.NewInt(math.MinInt64), big.NewInt(math.MinInt64)), // varintvalue
```

Comment:
Did you mean `big.NewInt(0).Sub(big.NewInt(math.MinInt64), big.NewInt(math.MaxInt64))`? Because `big.NewInt(0).Sub(big.NewInt(math.MinInt64), big.NewInt(math.MinInt64))` is `0`.

Reply:
Otherwise, this looks good to me.

Reply:
Fixed.